_specs/ecip-1077.md

```diff
@@ -0,0 +1,102 @@
+---
+ecip: 1077
+title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
+author: Mr. Meows D. Bits
+discussions-to: TBD
+status: WIP
+type: Meta
+created: 2019-11-29
+---
+
+### Abstract
+
+Establishing _Fork_ type ECIPs, and their standards and processes.
+
+#### TL;DR
+
+Make ECIPs with `Type` _Fork_ a thing. _Fork_ ECIPs should follow this process:
+- A "shell" Fork ECIP is opened, fitting the [template provided](TODO). This ECIP __does not initially include `Activation Block Number` or `Features` specifications__.
+- While in `WIP` or `Draft` status, one or more change sets are proposed against this document (eg. via Github Pull Requests) __modifying BOTH `Activation Block Number` and `Features` specifications__.
+- ... Stuff happens; there are comments, emails, meetings, bribes, blogs, bragging, trolling, haranguing, arguing, debating, pondering, editing, compromising, constructive criticiziing, etc...
```

Comment:
Just posting a potential revision that I think might fit @sorpaas' intentions w/r/t to discussion at https://github.com/ethereumclassic/ECIPs/pull/224#discussion_r352308946.

```suggestion
- _Rough consensus_ (keyword: see [definition](https://github.com/ethereumclassic/ECIPs/blob/c87f1828bd3d13ae5505f229207b65079a2de915/_specs/ecip-1000.md#L205) and [uses](https://github.com/ethereumclassic/ECIPs/search?q=rough+consensus&unscoped_q=rough+consensus)) is achieved and a single change set it explicitly approved by [ALL acting ECIP Editors](https://github.com/ethereumclassic/ECIPs/blob/c87f1828bd3d13ae5505f229207b65079a2de915/_specs/ecip-1000.md#ecip-editors). 
```